src/main/kotlin/traffic_simulation/Vehicle.kt

```diff
@@ -1,6 +1,6 @@
 package traffic_simulation
 
-class Vehicle(val capacityFactor: Double){
+class Vehicle(val id : Int , val wannaDrive : Boolean , val capacityFactor: Double = 1.0){
```

Comment:
Ich glaube der Grundgedanke von `wannaDrive` ist, und ich kann mir vorstellen dass @RF-Buerky  und ich da den gleichen Gedanken hatten (hatte ja dieses Attribut in meinem ersten Entwurf vor der Verschlankung), dass es auch Autos geben kann, die nicht fahren wollen. Dies entspricht meiner Meinung nach der Realität: ein Existierendes Auto steht auf dem Parkplatz `wannaDrive = false`. Wenn weitere Zeitschritte simuliert werden sollen, dann ließe sich für jeden Schritt ein Boolean aus der CSV einlesen; man erhielte somit quasi einen Fahrplan.